images/all-in-one/Dockerfile

```diff
@@ -1,13 +1,13 @@
-ARG GOLANG_VERSION=1.20.6
-ARG IBM_CLOUD_VERSION=2.18.0
-ARG TERRAFORM_PROVIDER_IBM_VERSION=1.55.0
-ARG TERRAFORM_PROVIDER_NULL_VERSION=3.2.1
+ARG GOLANG_VERSION=1.22.4
+ARG IBM_CLOUD_VERSION=2.26.1
+ARG TERRAFORM_PROVIDER_IBM_VERSION=1.66.0
+ARG TERRAFORM_PROVIDER_NULL_VERSION=3.2.2
 FROM golang:$GOLANG_VERSION AS builder
 
 RUN go install rsc.io/goversion@v1.2.0
 
 ARG TF_VERSION
-ENV TF_VERSION ${TF_VERSION:-v1.5.4}
+ENV TF_VERSION ${TF_VERSION:-v1.8.5}
```

Comment:
just be cautious with the version you want to refer here(keep in mind about the licence change!)

Reply:
Agree, BSL is the license is picked for releases - 1.6.0+.

I'll change the version to the latest available patch version in the 1.5.X release, which uses MPL v2.0 (I see 1.5.7 to be the last patch release). Please let me know if this should be ok. 

Thanks for pointing @mkumatag!

ref: https://github.com/hashicorp/terraform/commit/b145fbcaadf0fa7d0e7040eac641d9aef2a26433#comments

Reply:
Yes, the TF Version `1.5.7` worked fine.
I built the image locally and tested it.
[periodic-kubernetes-containerd-conformance-test-ppc64le](https://prow.ppc64le-cloud.cis.ibm.net/view/gs/ppc64le-kubernetes/logs/test-periodic-kubernetes-containerd-conformance-test-ppc64le/1804246043034914816) and [periodic-kubernetes-crio-node-throughput-perf-test-ppc64le](https://prow.ppc64le-cloud.cis.ibm.net/view/gs/ppc64le-kubernetes/logs/test-periodic-kubernetes-crio-node-throughput-perf-test-ppc64le/1804241857450872832) are the jobs that ran successfully with latest all-in-one image.

@kishen-v you may submit the change to modify `TF_VERSION`. Once this PR merges, the job postsubmit-allinone-job would build and push latest all-in-one image. We may later modify this in all other jobs.
@mkumatag

Reply:
Oh! I see tags changed to 0.7 for all jobs in this PR itself.
Once `TF_VERSION` is changed, this can merge.

Reply:
Hi @Rajalakshmi-Girish,

I've updated the `TF_VERSION` to 1.5.7.
Thanks!